pkg/aaparser/aaparser_test.go

```diff
@@ -71,3 +86,33 @@ Copyright 2009-2012 Canonical Ltd.
 		}
 	}
 }
+
+func requiresRoot(t *testing.T) {
+	if !rootEnabled {
+		t.Skip("skipping test that requires root")
+		return
+	}
+	require.Equal(t, 0, os.Getuid(), "This test must be run as root.")
+}
+
+func TestGetVersion(t *testing.T) {
+	output, err := cmd("", "--version")
+	require.NotNil(t, output)
+	version, err := GetVersion()
+	require.NoError(t, err)
+	require.NotEqual(t, -1, version)
+}
+
+func TestLoadProfile(t *testing.T) {
```

Comment:
This unit test invokes the external program `apparmor_parser`, which I believe has side effects on the system's apparmor profiles. I don't think a unit test should depend on external binaries, and it especially shouldn't reconfigure the system where it's run.

Finally, this test isn't actually checking that the profile was loaded correctly.

I think apparmor should be tested as part of integration tests, and I believe there is already coverage for this.